### ISSUE TITLE
Duplicates in machine_temperature_system_failure.csv

### ISSUE THREAD

**Issue by cansubasak** [numenta/NAB#376]
Duplicates with different observation values are as below. 

<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>value</th>
    </tr>
    <tr>
      <th>timestamp</th>
      <th></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>2014-01-07 02:00:00</th>
      <td>94.423406</td>
    </tr>
    <tr>
      <th>2014-01-07 02:05:00</th>
      <td>94.698730</td>
    </tr>
    <tr>
      <th>2014-01-07 02:10:00</th>
      <td>95.332824</td>
    </tr>
    <tr>
      <th>2014-01-07 02:15:00</th>
      <td>95.079199</td>
    </tr>
    <tr>
      <th>2014-01-07 02:20:00</th>
      <td>94.881208</td>
    </tr>
    <tr>
      <th>2014-01-07 02:25:00</th>
      <td>94.563961</td>
    </tr>
    <tr>
      <th>2014-01-07 02:30:00</th>
      <td>93.430922</td>
    </tr>
    <tr>
      <th>2014-01-07 02:35:00</th>
      <td>93.729663</td>
    </tr>
    <tr>
      <th>2014-01-07 02:40:00</th>
      <td>93.192987</td>
    </tr>
    <tr>
      <th>2014-01-07 02:45:00</th>
      <td>93.967871</td>
    </tr>
    <tr>
      <th>2014-01-07 02:50:00</th>
      <td>93.397374</td>
    </tr>
    <tr>
      <th>2014-01-07 02:55:00</th>
      <td>92.855999</td>
    </tr>
    <tr>
      <th>2014-01-07 02:00:00</th>
      <td>94.139723</td>
    </tr>
    <tr>
      <th>2014-01-07 02:05:00</th>
      <td>94.111970</td>
    </tr>
    <tr>
      <th>2014-01-07 02:10:00</th>
      <td>94.638723</td>
    </tr>
    <tr>
      <th>2014-01-07 02:15:00</th>
      <td>93.270907</td>
    </tr>
    <tr>
      <th>2014-01-07 02:20:00</th>
      <td>93.890249</td>
    </tr>
    <tr>
      <th>2014-01-07 02:25:00</th>
      <td>93.396627</td>
    </tr>
    <tr>
      <th>2014-01-07 02:30:00</th>
      <td>94.199300</td>
    </tr>
    <tr>
      <th>2014-01-07 02:35:00</th>
      <td>94.125420</td>
    </tr>
    <tr>
      <th>2014-01-07 02:40:00</th>
      <td>93.530827</td>
    </tr>
    <tr>
      <th>2014-01-07 02:45:00</th>
      <td>92.784720</td>
    </tr>
    <tr>
      <th>2014-01-07 02:50:00</th>
      <td>93.254724</td>
    </tr>
    <tr>
      <th>2014-01-07 02:55:00</th>
      <td>93.656042</td>
    </tr>
  </tbody>
</table>

**Comment by subutai** [comment]
These are not duplicates. They are spaced 5 minutes apart.

**Comment by cansubasak** [comment]
Please check again. Timestamps for the first row (_2014-01-07 02:00:00)_ and the twelfth row (_2014-01-07 02:00:00_) are same. There are 24 observation points between 2014-01-07 02:00:00 and 2014-01-07 02:55:00. (namely for 1 hour). I coincidentally realized that there are 300 data points for the day _2014-01-07_ .

**Comment by subutai** [comment]
Ah, thank you for pointing out the specifics. We had not noticed that before. Did you see if that happened elsewhere at all?

**Comment by cansubasak** [comment]
No, I checked all points. For the other days it's correct (with 5 min time steps, without any duplicates).
